### ISSUE TITLE
Add Lotrek skin

### ISSUE THREAD

**Issue by astagi** [lotrekagency/lotrekface#1]

**Comment by PurgePJ** [comment]
Can you please explain more in detail what you need?

**Comment by astagi** [comment]
Yes @PurgePJ I'd like to have a skin for my pebble with Lotrek Logo (https://cdn.filestackcontent.com/3VKYGlSgSnu0xwIcS7nc) with all the arrow of the "K" moving (one for seconds, one for minutes and one for hours, all with different sizes).. Me and my team stared experimenting with Pebble but now we haven't so much time to move forward this project
